app/src/renderer/components/staking/PageCandidates.vue

```diff
@@ -44,7 +44,7 @@ export default {
   computed: {
     ...mapGetters(['candidates', 'filters', 'shoppingCart']),
     pageTitle () {
-      return `Delegates (${this.candidatesNum} Candidates Selected)`
+      return `Delegate (${this.candidatesNum} Candidates Selected)`
```

Comment:
I don't think "Candidates Selected" belongs in the page title.

Reply:
Let's put it in the label for the Bond button instead.

Reply:
Hmm, let's keep this here until we design a new component to put this in.